### DISCUSSION TITLE
Option for a system administrator to view the chat correspondence of the users

### DISCUSSION THREAD

**Discussion by start-life** [FlowiseAI/Flowise#1530] [General]
I wonder if anyone knows how I can see conversations people have had with my bot?

**Comment by dkindlund** [comment]
You need to connect a 3rd party Analytic provider to examine the chat logs:
https://docs.flowiseai.com/using-flowise/analytic

**Comment by HenryHengZJ** [comment]
You can see all the messages from UI as well: https://docs.flowiseai.com/integrations/memory/long-term-memory#message-api

**Reply by dkindlund** [reply]
Hey @HenryHengZJ , I see the API URL for this in the documentation -- but what is the URL for the actual UI that you should in the bottom screenshot on this page?
https://docs.flowiseai.com/integrations/memory/long-term-memory#message-api
